cypress/support/component.ts

```diff
@@ -30,3 +30,15 @@ import { mount } from 'cypress/react18';
 Cypress.Commands.add('mount', mount);
 // Example use:
 // cy.mount(<MyComponent />)
+
+//@ts-ignore
+const check = components => {
```

Comment:
This could be typed as `ReactNode`, no? If not I rather have `any`

Reply:
The type certainly is ReactNode, but Cypress is throwing an Unexpected token error when the type is included. I'm leaving the exclusion until we can figure out how to handle the error.